apps/services/user-profile/src/app/user-profile/userProfile.controller.ts

```diff
@@ -67,6 +70,12 @@ export class UserProfileController {
     description: 'The nationalId of the application to update.',
     allowEmptyValue: false,
   })
+  @ApiQuery({
+    name: 'actorRequest',
+    required: false,
+    type: 'boolean',
+    description: 'Is the user requesting the actor information?',
+  })
```

Comment:
Forgot to remove the swagger decorator? Don't forget to remove the `ApiQuery` import when you remove this.

Reply:
@saevarma - This should be updated now. I was hurrying a little too much there.